four-node-cluster/run-performance-tests.sh

```diff
@@ -0,0 +1,281 @@
+#!/bin/bash -e
```

Comment:
can't we reduce the duplicate here?

We can move the test_scenarios to a common place and reuse it

Reply:
+1

Reply:
Addressed by https://github.com/wso2/performance-is/pull/128/commits/20e9c6ecdc067d08d3d4dade6fcb8d9f2a6e9d37